src/DerConverter/AssemblyInfo.cs

```diff
@@ -1,6 +1,7 @@
-﻿using System.Reflection;
+﻿using System;
```

Comment:
Hmm, AssemblyInfo is a left-over from the old project structure. I think this setting needs to be moved to https://github.com/huysentruitw/pem-utils/blob/master/src/DerConverter/DerConverter.csproj